components/mastodon/actions/post-multiple-statuses/post-multiple-statuses.mjs

```diff
@@ -0,0 +1,84 @@
+import mastodon from "../../mastodon.app.mjs";
+import { VISIBILITY_OPTIONS } from "../../common/constants.mjs";
+import { ConfigurationError } from "@pipedream/platform";
+
+export default {
+  key: "mastodon-post-multiple-statuses",
+  name: "Post Multiple Statuses",
+  description: "Publish multiple statuses with the given parameters. [See the documentation](https://docs.joinmastodon.org/methods/statuses/#create)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mastodon,
+    statuses: {
+      type: "string[]",
+      label: "Statuses",
+      description: "Array of status to be published in sequence, each status must be less than 500 characters long.",
+    },
+    inReplyToId: {
+      type: "string",
+      label: "In Reply To ID",
+      description: "ID of the status being replied to, if the status is a reply.",
+      optional: true,
+    },
+    sensitive: {
+      type: "boolean",
+      label: "Sensitive",
+      description: "Mark the status and attached media as sensitive? Defaults to false.",
```

Comment:
```suggestion
      description: "Mark the status and attached media as sensitive? Defaults to false.",
```

Reply:
Removed question mark to follow Google's Developer docs style guide (Avoid using questions in descriptions)